java/integTest/src/test/java/glide/TestUtilities.java

```diff
@@ -360,6 +360,23 @@ public static String createLuaLibWithLongRunningFunction(
                 .replace("$libName", libName);
     }
 
+    /**
+     * Create a lua script which runs an endless loop up to timeout sec.<br>
+     * Execution takes at least 5 sec regardless of the timeout configured.
+     */
+    public static String createLongRunningLuaScript(int timeout, boolean readOnly) {
+        String script =
+                "  local started = tonumber(redis.pcall('time')[1])\n"
+                        + "  while (true) do\n"
+                        + "    local now = tonumber(redis.pcall('time')[1])\n"
+                        + "    if now > started + $timeout then\n"
+                        + "      return 'Timed out $timeout sec'\n"
+                        + "    end\n"
+                        + "  end\n"
+                        + (readOnly ? "flags={ 'no-writes' }\n" : "");
```

Comment:
script has no flags, you need to write (or modify) a key

Reply:
realized this, will change